rs-publish.py

```diff
@@ -87,10 +87,22 @@ def get_file_content(self, commitish, path):
 to_update = []
 for r in source_records:
     record = records_by_id.pop(r["id"], None)
+
+    # Record does not exist, create it.
     if record is None:
         to_create.append(r)
-    elif not r == record:
+        continue
+
+    # Record exists, check if it matches the record from the repo.
+    # RemoteSettings includes meta-data fields which would always cause a
+    # mismatch when comparing records. Remove them before compare.
+    del record["last_modified"]
```

Comment:
Would it be better to delattr() or is del adequate?

I wanna make sure that there's no shadow attr persisting before garbage collection that could cause this update error to persist.

Page for context: https://www.geeksforgeeks.org/delattr-del-python/

Reply:
I think `del` is fine because we don't reference this key anywhere else. It's also faster.